chanbackup/backupfile_test.go

```diff
@@ -188,7 +190,7 @@ func assertMultiEqual(t *testing.T, a, b *Multi) {
 func TestExtractMulti(t *testing.T) {
 	t.Parallel()
 
-	keyRing := &mockKeyRing{}
+	keyRing := &lnencrypt.MockKeyRing{}
```

Comment:
Can't we move to the `mock.MockKeyRing` here directly? Instead of creating a new one and then moving away from that in a later commit?

Reply:
Ah yes so I think I was attempting to split out the mock key change into a new PR as requested here https://github.com/lightningnetwork/lnd/pull/4458#discussion_r778065722

But perhaps it'd be better to resquash it into one commit? Thoughts?